.github/CODEOWNERS

```diff
@@ -5,15 +5,21 @@
 /.github                                    @justinahinon @felixarntz
 
 # Focus: Images
-/modules/webp-uploads                       @adamsilverstein
-/tests/modules/webp-uploads                 @adamsilverstein
+/modules/images                             @adamsilverstein
+/tests/modules/images                       @adamsilverstein
```

Comment:
Maybe in addition to those two locations, we should also add `/tests/testdata/modules/{focus}` here for each module? That's the third focus-specific location, which is more minor but probably still worth capturing here for completeness.

Reply:
That makes sense, yes.

One another thing @felixarntz. Should I add you as owner for `/tests/admin`?